test/jdk/java/awt/TrayIcon/DisposeInActionEventTest/DisposeInActionEventTest.java

```diff
@@ -100,30 +93,30 @@ public void start() {
         g.fillRect(6, 6, 20, 20);
         g.dispose();
 
-        systemTray = SystemTray.getSystemTray();
-        trayIcon = new TrayIcon(img);
+        SystemTray systemTray = SystemTray.getSystemTray();
+        TrayIcon trayIcon = new TrayIcon(img);
         trayIcon.setImageAutoSize(true);
         trayIcon.addActionListener(ev -> {
-            Sysout.println(ev.toString());
+            textArea.append(ev.toString() + "\n");
```

Comment:
```suggestion
            textArea.append(ev + "\n");
```
`.toString()` is redundant.